src/views/BrokeringRuns.vue

```diff
@@ -175,8 +175,31 @@ async function setEComStore(event: CustomEvent) {
   emitter.emit("dismissLoader")
 }
 
-function getScheduleFrequency(cronExp: string) {
-  return Object.entries(cronExpressions).find(([description, expression]) => expression === cronExp)?.[0] || "-"
+function getScheduleFrequency(brokeringGroupObj: any) {
+  const foundDescription = Object.entries(cronExpressions).find(
+    ([description, expression]) => expression === brokeringGroupObj.cronExpression
+  )?.[0];
+
+  if (foundDescription) {
+    return foundDescription;
+  }
+
+  if (brokeringGroupObj.cronDescription) {
```

Comment:
No need to add this logic for processing the description instead simply use `cronstrue.toString` function to convert expression into string